src/i965_drv_video.c

```diff
@@ -6720,6 +6720,7 @@ i965_Terminate(VADriverContextP ctx)
     int i;
 
     if (i965) {
+#if HAVE_HYBRID_CODEC
```

Comment:
Is this really necessary since we check for i965->wrapper_pdrvctx.  I know, it's a compile-time check vs. runtime check... but we're in i965_Terminate and don't think this change will give us much gain.  If we do this compile-time check, why not change it everywhere else we check for i965->wrapper_pdrvctx?